src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/CaseloadService.kt

```diff
@@ -16,17 +16,13 @@ import java.time.LocalDate
 class CaseloadService(
   private val prisonerSearchApiClient: PrisonerSearchApiClient,
   private val releaseDateService: ReleaseDateService,
-  private val licenceCreationService: LicenceCreationService,
 ) {
 
   fun getPrisonersByNumber(nomisIds: List<String>): List<CaseloadItem> {
     val prisoners = prisonerSearchApiClient.searchPrisonersByNomisIds(nomisIds)
 
-    val prisonersToLicenceKinds = prisoners.associate {
-      it.prisonerNumber to licenceCreationService.determineLicenceKind(it)
-    }
```

Comment:
this is what @justin-gardiner just put in yesterday to get PRRD cases to appear in the CRD list 😂